srcpkgs/gnome-weather/template

```diff
@@ -1,22 +1,21 @@
 # Template file for 'gnome-weather'
 pkgname=gnome-weather
-version=3.26.0
+version=3.32.0
 revision=1
-lib32disabled=yes
-build_style=gnu-configure
-configure_args="--disable-schemas-compile"
-hostmakedepends="pkg-config intltool itstool gnome-doc-utils glib-devel
- $(vopt_if gir gobject-introspection)"
+build_helper="gir"
+build_style=meson
+hostmakedepends="pkg-config itstool glib-devel gjs"
 makedepends="gtk+3-devel gjs-devel libgweather-devel geoclue2-devel"
 depends="desktop-file-utils"
```

Comment:
missing gjs, geoclue2, libgweather, gnome-desktop on the dependencies.

There are no shlibs that can be used to find dependencies properly. -1 for gjs

Reply:
Merged manually with these added dependencies or the package doesn't start when installed without having the full GNOME stack installed.